templates/index.php

```diff
@@ -2,14 +2,13 @@
 use OCA\News\Plugin\Client\Plugin;
 
 script('news', [
-    'vendor/js-url/url.min',
-    'vendor/es6-shim/es6-shim.min',
-    'vendor/angular/angular.min',
-    'vendor/angular-animate/angular-animate.min',
-    'vendor/angular-route/angular-route.min',
-    'vendor/angular-sanitize/angular-sanitize.min',
-    'vendor/moment/min/moment-with-locales.min',
-    'vendor/masonry/dist/masonry.pkgd.min',
+    'node_modules/es6-shim/es6-shim.min',
```

Comment:
Just quick question: why dont we simply concat all scripts into one bundle? Then we dont need to ship anything other than the app.min.js file and can blacklist node_modules again